admin/views/service-manager/upload-pupil-census.ejs

```diff
@@ -0,0 +1,45 @@
+<div class="grid-row">
+    <div class="column-one-half">
+        <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+    </div>
+    <div class="column-one-half">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<header class="page-header">
+    <h1 class="heading-xlarge">Upload pupil census</h1>
+</header>
+
+<div class="column-two-thirds">
+    <br>
+    <form action="#" enctype="multipart/form-data" method="post" name="upload-pupil-census" id="upload-pupil-census">
+        <div class="form-group">
+            <div class="font-xsmall">
+                <input type="file" name="csvTemplateFile" id="file-upload" accept="text/csv"
+                       aria-label="Choose a file to upload"/>
+            </div>
+            <input type="reset" class="link remove-file" id="removeUploadedFile" value="Remove file"
+                   aria-label="Remove uploaded file."/>
+        </div>
+        <div class="form-buttons">
+            <input type="submit" id="upload-form-submit" class="button" value="Upload" disabled="disabled"/>
+            <a href="/service-manager/home" class="button button-secondary">Cancel</a>
+        </div>
+    </form>
+
+    <table class="spacious reduced-top-margin" id="pupilCensusList" role="listbox">
```

Comment:
Why is this in a table?  I don't see tabular data.

Reply:
@jon-shipley updated